systemtest/src/main/java/io/strimzi/systemtest/resources/KubernetesResource.java

```diff
@@ -307,6 +312,84 @@ public static Service deployBridgeNodePortService(String bridgeExternalService,
             .endSpec().build();
     }
 
+    public static <T extends HasMetadata> NetworkPolicy applyNetworkPolicySettingsForResource(T resource, String name) {
+        NetworkPolicy networkPolicy = new NetworkPolicyBuilder()
+                .withNewApiVersion("networking.k8s.io/v1")
+                .withNewKind("NetworkPolicy")
+                .withNewMetadata()
+                    .withName(resource.getMetadata().getName() + "-allow")
+                .endMetadata()
+                .withNewSpec()
+                    .addNewIngress()
+                        .addNewPort()
+                            .withNewPort(8083)
+                            .withNewProtocol("TCP")
+                        .endPort()
+                        .addNewPort()
+                            .withNewPort(9404)
+                            .withNewProtocol("TCP")
+                        .endPort()
+                        .addNewPort()
+                            .withNewPort(8080)
+                            .withNewProtocol("TCP")
+                        .endPort()
+                      .endIngress()
+                    .withNewPodSelector()
+                        .addToMatchLabels("strimzi.io/cluster", resource.getMetadata().getName())
+                        .addToMatchLabels("strimzi.io/kind", resource.getKind())
+                        .addToMatchLabels("strimzi.io/name", name)
+                    .endPodSelector()
+                    .withPolicyTypes("Ingress")
+                .endSpec()
+                .build();
+
+        deleteLater(kubeClient().getClient().network().networkPolicies().inNamespace(ResourceManager.kubeClient().getNamespace()).createOrReplace(networkPolicy));
+        return networkPolicy;
+    }
+
+    public static NetworkPolicy allowNetworkPolicies(String namespace) {
```

Comment:
Again, the naming here might be confusing since it doesn't deny or allow network policies. It basically just defines their default behaviour if they are already enabled. If they are disabled, this will do nothing.